.travis.yml

```diff
@@ -33,9 +33,9 @@ install:
 before_script:
   - sh -c "if [ '$TEST_SUITE' = 'configurator' ]; then ./Test/Integration/setup-magento.sh $MAGE_VERSION; fi"
 script:
-  - sh -c "if [ '$TEST_SUITE' = 'phpcs' ]; then php vendor/bin/phpcs --standard=PSR2 Model/ Console/ Test/ Component/ Api/ Exception/; fi"
```

Comment:
Hi @bartoszherba,

Would it be possible to include these back in? It would be nice to ensure our tests are written neatly.

Thanks :)

Reply:
Ok, I just did.